lib/browser.js

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import bedrock from 'bedrock';
+import 'bedrock-express';
+import onHeaders from 'on-headers';
+
+bedrock.events.on('bedrock-express.init', app => {
+  app.use((req, res, next) => {
+    onHeaders(res, function() {
+      const setCookie = res.getHeader('set-cookie');
+      const userAgent = req.headers['user-agent'];
+      if(!setCookie) {
+        return;
+      }
+      const hasSafari = _hasSafari(userAgent);
+      let isOldSafariVersion = false;
+      if(hasSafari) {
+        isOldSafariVersion = _isOldSafariVersion(userAgent);
+      }
```

Comment:
```suggestion
      if(!(_hasSafari(userAgent) && _isOldSafariVersion(userAgent))) {
        return;
      }
```

Reply:
Is it faster to check for `SameSite=None` or check the user agent? Whichever is faster to get us short-circuiting is what we should do first.